internal/acceptance_test/resource_host_test.go

```diff
@@ -93,11 +116,18 @@ locals  {
 		untagged = `""`
 	}
 
+	name := "test%v"
+	if isAsync {
+		name = fmt.Sprintf(name, "async")
+	} else {
+		name = fmt.Sprintf(name, "sync")
```

Comment:
Make the names "testAsync", "testSync"? I think it'd be easier to read then "testasync" and "testsync".
```suggestion
		name = fmt.Sprintf(name, "Async")
	} else {
		name = fmt.Sprintf(name, "Sync")
```